src/commands/openapi/upload.ts

```diff
@@ -40,6 +40,14 @@ export default class OpenAPIUploadCommand extends BaseCommand<typeof OpenAPIUplo
         'If set, file overwrites will be made without a confirmation prompt. This flag can be a useful in automated environments where prompts cannot be responded to.',
       hidden: true,
     }),
+    'legacy-id': Flags.string({
+      summary: 'The legacy ID for your API definition.',
+      description:
+        'This is only used for legacy rdme CLI workflows and only applies if your project, and this API definition, predates ReadMe Refactored. We consider this value to be deprecated and recommend against relying on it going forward.',
```

Comment:
could use some more massaging but i think we should point them towards `slug` being our recommended argument 

```suggestion
        'This is only used for legacy rdme CLI workflows and only applies if your project, and this API definition, predates ReadMe Refactored. We consider argument value to be deprecated and recommend against relying on it going forward, instead prefer using `slug`.',
```

Reply:
how about this?

```suggestion
        'This is only used for legacy `rdme` CLI workflows and only applies if your project, and this API definition, predate ReadMe Refactored. This flag is considered deprecated and we recommend using `--slug` instead.',
```